java/arcs/core/storage/DriverFactory.kt

```diff
@@ -102,4 +109,19 @@ interface DriverProvider {
     suspend fun removeAllEntities() = Unit
 
     suspend fun removeEntitiesCreatedBetween(startTimeMillis: Long, endTimeMillis: Long) = Unit
+
+    /**
+     * @param inMemory if true, return count of entities stored in-memory, otherwise return count
+     * of entities stored on-disk.
+     */
+    suspend fun getEntitiesCount(inMemory: Boolean): Long = 0
+
+    /**
+     * @param inMemory if true, return bytes stored in-memory, otherwise return bytes
+     * stored on-disk.
+     */
+    suspend fun getStorageSize(inMemory: Boolean): Long = 0
+
+    /** Returns if the storage is too large. */
```

Comment:
please add more details on what it means to be too large

Reply:
Done.